components/narrator-card.tsx

```
@@ -1,14 +1,110 @@
+  ArrowUpRightIcon,
+  BabyIcon,
+  CircleHelpIcon,
+  MapPinIcon,
+  SkullIcon,
+} from "lucide-react";
 import Link from "next/link";
 
-const formatDate = (date: string, type: "hijri" | "gregorian") => {
-  const calendar = type === "hijri" ? "هجريا" : "ميلاديا";
-  return (date === null ? "غير معلوم" : `${date}`) + ` (${calendar})`;
+const Date = ({ value }: { value: string | undefined }) => {
```

Biome (1.9.4)

[error] Do not shadow the global "Date" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)